self-hosted/upgrades/upgrade-docker.md

```diff
@@ -35,123 +35,204 @@ mounts, or bind mounts.
 
 <Procedure>
 
-### Determining the mount point type
-
-1.  Work out what type of mount your Docker container uses by running this
-    command, which returns either `volume` or `bind`:
+1.  Find the mount type used by your Docker container: 
 
     ```bash
     docker inspect timescaledb --format='{{range .Mounts }}{{.Type}}{{end}}'
     ```
+    This returns either `volume` or `bind`.
 
-1.  Get the current name or mount path with this command, and record it to use
-    when you perform the upgrade. Make sure you copy the correct command, based
-    on your mount point type.
+1.  Note the volume or bind used by your container:
 
     <Terminal>
 
-    <tab label='Volume mount'>
+    <tab label='Volume'>
 
     ```bash
     docker inspect timescaledb --format='{{range .Mounts }}{{.Name}}{{end}}'
+    ```
+    Docker returns the `<volume ID>`. You see something like this: 
+
+    ```
     069ba64815f0c26783b81a5f0ca813227fde8491f429cf77ed9a5ae3536c0b2c
     ```
 
     </tab>
 
-    <tab label='Bind mount'>
+    <tab label='bind'>
 
     ```bash
     docker inspect timescaledb --format='{{range .Mounts }}{{.Source}}{{end}}'
+    ```
+
+    Docker returns the `<bind path>`. You see something like this: 
```

Comment:
I'm not sure all this is specifically required? Possibly just something about making sure the volume bind/mount and PGDATA are kept 100% consistent between running and upgrading? They could be using the image for k8s, docker swarm, docker compose, docker, podman.... it might be a mess to go too far into the weeds on things other than just making sure they always pick the same mount and set PGDATA to point to it.